zingolib/src/wallet/data.rs

```diff
@@ -84,6 +84,7 @@ impl BlockData {
         Self { ecb, height }
     }
 
+    #[cfg(not(feature = "sync"))]
```

Comment:
Why is creating a BlockData from a CompactBlock different between sync implementations? Is it because BlockData aren't used in peppersync? Is that known yet, with all the todo!s still in existence?

Reply:
yes pepper-sync has a new struct for block data called WalletBlock